src/ape_console/_cli.py

```diff
@@ -24,6 +32,57 @@ def cli(cli_ctx, network):
     return console(verbose=verbose)
 
 
+def import_extras_file(file_path) -> ModuleType:
+    """Import a module"""
+    loader = SourceFileLoader(file_path.name[:-3], str(file_path))
+    spec = spec_from_loader(loader.name, loader)
+
+    assert spec is not None
+
+    module = module_from_spec(spec)
+    loader.exec_module(module)
+
+    return module
+
+
+def load_console_extras(namespace: dict[str, Any]) -> dict[str, Any]:
+    """load and return namespace updates from ape_console_extras.py  files if
+    they exist"""
+    global_extras = config.DATA_FOLDER.joinpath(CONSOLE_EXTRAS_FILENAME)
+    project_extras = config.PROJECT_FOLDER.joinpath(CONSOLE_EXTRAS_FILENAME)
+
+    for extras_file in [global_extras, project_extras]:
+        if extras_file.is_file():
```

Comment:
To save an elongated indent, you can do this:

```suggestion
        if not extras_file.is_file():
            continue
```

and then de-indent everything below.